app/assets/stylesheets/custom.scss

```diff
@@ -133,10 +133,6 @@
   text-align: center;
 }
 
-a:visited {
-  color: #EEF5DB;
-}
-
```

Comment:
let's add this back in at some point; we can just change it to be a more readable color